package.json

```diff
@@ -13,24 +13,30 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "clean": "rm -rf lib es",
+    "lint": "standard --verbose | snazzy",
     "build:commonjs": "BABEL_ENV=commonjs babel --quiet src --out-dir lib",
     "build:es": "BABEL_ENV=es babel --quiet src --out-dir es",
     "build": "npm run clean && npm run build:es && npm run build:commonjs",
-    "prepublish": "npm run clean && npm run build"
+    "prepublish": "npm run lint && npm run clean && npm run build"
   },
   "dependencies": {
-    "glamor": "^2.20.20"
+    "glamor": "2.20.24"
```

Comment:
I'll change it but FYI using a caret is making the bold assumption that any library you rely on will strongly follow semver guidelines, which is sadly not always the case. IMHO is always better to keep it deterministic.